plugins/extractors/mysql/mysql_test.go

```diff
@@ -15,10 +15,10 @@ import (
 	_ "github.com/go-sql-driver/mysql"
 	"github.com/odpf/meteor/plugins"
 	"github.com/odpf/meteor/plugins/extractors/mysql"
-	"github.com/odpf/meteor/plugins/testutils"
 	"github.com/odpf/meteor/proto/odpf/assets"
 	"github.com/odpf/meteor/proto/odpf/assets/common"
 	"github.com/odpf/meteor/proto/odpf/assets/facets"
+	"github.com/odpf/meteor/testutils"
```

Comment:
@StewartJingga Let's rename this to just `test` package.

Reply:
Any reason we are not including it in `utils` package we already have in root??

Reply:
@ravisuhag the `test` folder breaks `make test` command and the workaround is `make -B test`.

Should we keep the `testutils` package or maybe put `test` package into `utils`?

Reply:
@StewartJingga more context on why make is breaking with that?

Reply:
If you have `test` folder while running `make test`, it would refer to that folder instead of the command you specified in Makefile.

But I just found a fix for that, we can put `test` to always be a command like this

```
# Makefile

.PHONY: test
```